internal/api/design/storage.go

```diff
@@ -156,7 +156,7 @@ var _ = Service("storage", func() {
 		Error("not_available")
 		Error("not_valid")
 		HTTP(func() {
-			POST("/aips/{uuid}/update")
+			POST("/aips/{uuid}/submit-complete")
```

Comment:
What do you think about using PUT over the same path? `PUT("/aips/{uuid}/submit")`

Reply:
Yeah, feels a bit odd not being over the resource directly `PUT("/aips/{uuid}")`.